Monika After Story/game/script-compliments.rpy

```diff
@@ -530,3 +530,72 @@ label mas_compliment_bestgirl:
     m 5ekbfa "But if you ask me, you made the right choice."
     m 5hubfa "...And I'll be forever grateful that you did~"
     return
+    
+    init 5 python:
+        addEvent(
+            Event(
+                persistent._mas_compliments_database,
+                eventlabel="mas_compliment_thinking_of_you",
+                prompt="... I'm always thinking about you!",
+                unlocked=True
+            ),
+            eventdb=store.mas_compliments.compliment_database
+        )
+
+label mas_compliment_thinking_of_you:
+    if not renpy.seen_label("mas_compliment_thinking_of_you_2"):
+        call mas_compliment_thinking_of_you_2
+    else:
+        call mas_compliment_thinking_of_you_3
+    return
+
+label mas_compliment_thinking_of_you_2:
+    m 1wubso "..."
+    m 1wubfb "You..."
+        
+    if mas_isMoniNormal:
+        m 2hubfb "You have no idea what that means to me, [player]."
+        
+    elif mas_isMoniUpset:
+        m "That you would say that..."
```

Comment:
This line doesn't really line up with the previous line.

Reply:
I apologize, I must make my leave. But I shall return once rested, and put my metaphorical back into it.